src/components/pages/about-us/impressions.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {
+  AboutUsImpressionTileSize,
+  ContentfulAboutUsImpression,
+} from '../../../types';
+import { SectionHeader } from '../../content/section-header/section-header';
+import { useTranslation } from 'gatsby-plugin-react-i18next';
+import styled from 'styled-components';
+import { GatsbyImage, getImage } from 'gatsby-plugin-image';
+import { up } from '../../support/breakpoint';
+
+interface ImpressionsProps {
+  impressions: ContentfulAboutUsImpression[];
+}
+
+const GalleryGrid = styled.div`
+  display: grid;
+  gap: 8px;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(auto-fill, 250px);
```

Comment:
So that solves the row count issue I highlighted on the previous draft by @timonsommer  🙌